MdePkg/Include/IndustryStandard/Acpi51.h

```diff
@@ -1823,6 +1823,19 @@ typedef struct {
   UINT32    EntryCount;
 } EFI_ACPI_5_1_EINJ_TRIGGER_ACTION_TABLE;
 
+///
+/// Windows ACPI Emulated devices Table
+///
+typedef struct {
+  EFI_ACPI_DESCRIPTION_HEADER    Header;
+  ///
+  /// Container of a bitmask of Windows behavior that this system requires
+  /// Bit 0 - RTC good

```

Comment:
Please add `#defines` for `BIT0` and `BIT1` to make the evaluation of the `EmulatedDeviceFlags` field easier to use and understand.

Here is an example to follow in same file. ACPI Spec has a table that defines the bits in the field of the structure. The .h file adds the #defines for each table entry.

https://github.com/tianocore/edk2/blob/23007f7ae2f9fd63540a4cebeba7fa40e8b543e2/MdePkg/Include/IndustryStandard/Acpi51.h#L522

Reply:
Done.